### PR TITLE
:seedling: [POC] [DNR] Test for adding mp to quickstart templates

### PR DESCRIPTION
**Pull request by killianmuldoon**
Signed-off-by: killianmuldoon <kmuldoon@vmware.com><!-- Thanks for sending a pull request! Here are some tips for you:

Testing for errors in https://github.com/kubernetes-sigs/cluster-api/pull/9393

### CODE REVIEW

**Comment by killianmuldoon** [comment]
pull-cluster-api-e2e-full-main

**Review by killianmuldoon — COMMENTED** [review]
/pull-cluster-api-e2e-full-main

File: test/framework/alltypes_helpers.go [hunk 1]
```diff
@@ -151,7 +151,7 @@ func DumpKubeSystemPodsForCluster(ctx context.Context, input DumpKubeSystemPodsF
 	podList.SetKind("Pod")
 	var listErr error
 	_ = wait.PollUntilContextTimeout(ctx, retryableOperationInterval, retryableOperationTimeout, true, func(ctx context.Context) (bool, error) {
-		if listErr = input.Lister.List(ctx, podList, client.InNamespace(metav1.NamespaceSystem)); listErr != nil {
```
Comment (killianmuldoon) [inline test/framework/alltypes_helpers.go:154]: @sbueringer @chrischdi WDYT about making a change like this anyway? We could just go through the known CAPI namespaces, but it makes sense to me to just dump all the pods in the workload clusters.

Comment (sbueringer) [reply test/framework/alltypes_helpers.go:154]: Yeah let's just dump all Pods. Feel free to open a separate PR. I just approved the collecting Nodes PR so you can built it on top of it.

Comment (chrischdi) [reply test/framework/alltypes_helpers.go:154]: aggree, dumping all pods would be a great thing esepcially for self-hosted tests :-)

**Comment by killianmuldoon** [comment]
/close 

going to close this as it's unnecessary now. Dumping pods is in #9441 and the commit that fixes https://github.com/kubernetes-sigs/cluster-api/pull/9393 is https://github.com/kubernetes-sigs/cluster-api/pull/9440/commits/4bc3ba3ba5e90d60431fc53d5cf6778212b247ef

**Pull request closed without merging**